packages/playground/src/components/setupSolid.ts

```diff
@@ -1,6 +1,6 @@
 import { languages } from 'monaco-editor';
 
-const solidTypes = import.meta.glob('/node_modules/{solid-js,csstype}/**/*.{d.ts,json}', { eager: true, as: 'raw' });
```

Comment:
`as: 'raw'` is deprecated in vite 5, in favor of `query: '?raw', import:"default"`